apps/emqx/test/emqx_cm_SUITE.erl

```diff
@@ -219,7 +219,14 @@ t_open_session_race_condition(_) ->
         bug_triggered ->
             ok;
         Winner when is_pid(Winner) ->
-            ?assertMatch([_], ets:lookup(?CHAN_TAB, ClientId)),
+            ?assertMatch(
+                [_],
+                ?retry(
+                    _Interval = 100,
+                    _NTimes = 10,
+                    [_] = ets:lookup(?CHAN_TAB, ClientId)
```

Comment:
case was failed that 750/2000 channels are not cleaned, think having some retires should solve the problem.